analytics_framework/rag_system/setup_chroma_vector_db.py

```diff
@@ -58,18 +59,49 @@ def chromadb_processing(documents, ids, db_collection_name, what_to_query, resul
     return outcome
 
 
-if __name__ == '__main__':
-    documents, ids = yt_data_read_process("../intake/data/YoutubeCommentsDataSet.csv",
-                                          1000,
-                                          "Comment"
-                                          )
-    results = chromadb_processing(documents,
-                                  ids,
-                                  "YT_comments_new",
-                                  "My macbook broke in an accident",
-                                  )
-
-    # voila mix multiple query types to arrive at the desired outcome
-    print(f"The outcome for the query text \n {results}")
-    # just the 'documents' part is interesting to us, let's scoop it out
-    print(results["documents"])
+@click.command()
+@click.option(
+    "--query",
+    "-q",
+    default="My macbook broke in an accident",
```

Comment:
The default fallback is great. Could we have a message stating the same?

Example: 

`defaulting to "...." since no query provided`